features/retry.feature

```diff
@@ -570,3 +570,61 @@ Feature: Retry flaky tests
       And scenario "Failing" attempt 0 step "Given a failing step" has status "failed"
       And scenario "Failing" attempt 1 step "Given a failing step" has status "failed"
       And scenario "Passing" step "Given a passing step" has status "skipped"
+
+  Scenario: RerunFormatter does not report attempts that are retried
+    Given a file named "features/a.feature" with:
+      """
+      Feature:
+        Scenario: Flaky
+          Given a flaky step
+      """
+    Given a file named "lib/custom_formatter.js" with:
```

Comment:
HINT: It becomes more READABLE if you are using „And …“ (instead of: „Given …“) for the second and following given-steps.